dynawo/sources/Models/CPP/ModelNetwork/DYNModelTapChanger.h

```diff
@@ -66,20 +67,19 @@ class ModelTapChanger {
    * @return step associated to the index
    */
   inline const TapChangerStep& getStep(int key) const {
-    if (steps_.find(key) != steps_.end()) {
-      return steps_.find(key)->second;
-    } else {
+    try {
+      return steps_.at(key - getLowStepIndex());
+    } catch (...) {
```

Comment:
Generic catch should be limited, please put a specific error or a different way to catch this (if key >= steps_.size() for example)